Core/include/Acts/Digitization/PlanarModuleCluster.hpp

```diff
@@ -40,8 +40,7 @@ class PlanarModuleCluster : public Measurement<BoundIndices, 3> {
                       Base::CovarianceMatrix cov, double loc0, double loc1,
                       double t, std::vector<DigitizationCell> dCells,
                       const DigitizationModule* dModule = nullptr)
-      : Base(sourceLink, kIndices, Base::ParametersVector(loc0, loc1, t),
-             std::move(cov)),
+      : Base(sourceLink, kIndices, Base::ParametersVector(loc0, loc1, t), cov),
```

Comment:
why did it complain about the moves? not movable?

Reply:
I think "trivially copyable".